web-ui/src/components/feedback_request_card/feedback_request_subcard/FeedbackRequestSubcard.jsx

```diff
@@ -3,7 +3,7 @@ import { styled } from "@mui/material/styles";
 import PropTypes from "prop-types";
 import Grid from "@mui/material/Grid";
 import Typography from "@mui/material/Typography";
-import {Link} from "react-router-dom";
+import { Link } from "react-router-dom";
```

Comment:
If you don't want Prettier to add those spaces inside curly braces and you don't want strings to be delimited with double-quotes, you can create a `.prettierrc` file at the root of your app containing one or more of these settings:

```json
{
  "arrowParens": "avoid",
  "bracketSpacing": false,
  "htmlWhitespaceSensitivity": "ignore",
  "printWidth": 80,
  "singleQuote": true,
  "trailingComma": "none",
  "useTabs": false
}
```